src/kademlia/node.cpp

```diff
@@ -859,7 +858,9 @@ void node::incoming_request(msg const& m, entry& e)
 		bool scrape = false;
 		if (msg_keys[1] && msg_keys[1].int_value() != 0) noseed = true;
 		if (msg_keys[2] && msg_keys[2].int_value() != 0) scrape = true;
-		lookup_peers(info_hash, reply, noseed, scrape);
+		bool full = lookup_peers(info_hash, reply, noseed, scrape);
```

Comment:
I think this would be a good place to copy-paste the the8472's comment, or paraphrase